datasafe-test-storages/src/main/java/de/adorsys/datasafe/teststorage/WithStorageProvider.java

```diff
@@ -220,6 +225,17 @@ protected static StorageDescriptor cephVersioned() {
         );
     }
 
+    private static boolean skipCeph() {
+        String value = System.getProperty(SKIP_CEPH);
+        if (value == null) {
```

Comment:
Very complicated check why not:
return "true".equals(System.getProperty(SKIP_CEPH))

Reply:
this is something different. In my case it would work to do
-DSKIP_CEPH
yours would require
-DSKIP_CEPH=true
I prefere to have the first solution too.

Reply:
ok